engine/src/engine.ts

```diff
@@ -205,12 +214,22 @@ export default class Engine {
 
   /** Fix old options passed. To remove when legacy data is no more in database */
   sanitizeOptions() {
+    if (this.options.factionVariant == null) {
+      this.options.factionVariant = (process.env.VUE_APP_factionVariant ?? "standard") as FactionVariant;
```

Comment:
`VUE_APP_factionVariant` is a viewer variable (that you could use in `self-contained.ts`)

If you want, you could have something like:

``` 
Engine.defaults = {
  factionVariant: "standard"
}
```

and here:

```
   this.options.factionVariant = Engine.defaults.factionVariant;
```

and in `self-contained.ts`:

```
if (process.env.VUE_APP_factionVariant) {
  Engine.defaults.factionVariant = ...
}
```

But you could just change:

```
let engine = new Engine(["init 3 12"], { auction: true }, "5.6.10");
```

to

```
let engine = new Engine(["init 3 12"], { auction: true, factionVariant: "more-balanced" }, "5.6.10");
```

or


```
let engine = new Engine(["init 3 12"], { 
  auction: true, 
  factionVariant: process.env.VUE_APP_factionVariant ?? "standard"
}, "5.6.10");
```